en/flight_controller/pixhawk4.md

```diff
@@ -37,8 +37,7 @@
 * Other Characteristics:
   * Operating temperature: -40 ~ 85°c
 
-Additional information can be found in the [Pixhawk 4 Technical Data Sheet]().
-<!-- Later change to holybro wesbite (https://github.com/PX4/Hardware/blob/master/FMUv5/Pixhawk4-Datasheet.pdf). -->
+Additional information can be found in the [Pixhawk 4 Technical Data Sheet](http://www.holybro.com/manual/Pixhawk4-DataSheet.pdf).
```

Comment:
This link gives me `www.holybro.com refused to connect.` message

Reply:
It works fine for me. Can you check again? If it still doesn't work then I can add the link to the datasheet we have in github.

Reply:
Works now. They must have added it later.